includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -95,11 +95,74 @@ public function getData() {
 		return $data;
 	}
 
-	private function getHref( $hrefKey, $protocolRelative = false ) {
+	protected function getDomainForCityId( $cityId ) {
```

Comment:
I assume this was changed to `protected` in order to mock it in tests? I don't really like that as a pattern as it usually suggests your tests are too dependent on implementation details. In this case, I'd suggest on'y mocking the external dependencies and setting this and `getDevDomain` back to private since they are not being overridden by subclasses.

If we remove the `$cityId` per another comment, this might be even simpler.

Reply:
@grunny By doing this I would need to mock global variables which is really not encouraged either:
* `$wgServer`
* `$wgDevDomain`
* `$wgDevelEnvironment`
Basically I should use `WikiaBaseTest`.